src/Components/SmartComponents/Modals/OsExposureReportModal/OsExposureReportModal.js

```diff
@@ -180,7 +184,9 @@ const OsExposureReportModal = ({
                                                             !isMinorOsVersionSelected({ [osMinor]: minorValues })
                                                             && selectedOsVersions.length > 4
                                                         }
-                                                        onChange={() => toggleArrayItem({ [osMinor]: minorValues })}
+                                                        onChange={
+                                                            () => toggleArrayItem({ [osMinor]: { minor: minorValues.minor } })
+                                                        }
```

Comment:
Is it guaranteed that every version has at least the minor release lifecycle support? This can potentially break if no. Just want to make sure.